src/main/kotlin/no/nav/familie/ks/sak/kjerne/endretutbetaling/EndretUtbetalingAndelService.kt

```diff
@@ -59,23 +59,12 @@ class EndretUtbetalingAndelService(
             hentEndredeUtbetalingAndeler(behandling.id)
                 .filter { it.id != endretUtbetalingAndelRequestDto.id }
 
-        val gyldigTomEtterDagensDato =
-            beregnGyldigTomIFremtiden(
-                andreEndredeAndelerPåBehandling = andreEndredeAndelerPåBehandling,
-                endretUtbetalingAndel = endretUtbetalingAndel,
-                andelTilkjentYtelser = andelTilkjentYtelser,
-            )
-
         validerTomDato(
             tomDato = endretUtbetalingAndel.tom,
-            gyldigTomEtterDagensDato = gyldigTomEtterDagensDato,
+            gyldigTomEtterDagensDato = endretUtbetalingAndel.tom,
             årsak = endretUtbetalingAndel.årsak,
```

Comment:
Her burde `validerTom` oppdateres. Gir ikke mening å sende inn samme verdi for to ulike parametre sånn det er nå 😅